test/gateway/safe_core/get_state_root.js

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const SafeCore = artifacts.require("./SafeCore.sol");
+const MockMembersManager = artifacts.require('MockMembersManager.sol');
+const web3 = require('../../test_lib/web3.js');
+const BN = require('bn.js');
+
+contract('SafeCore.getStateRoot()', function (accounts) {
```

Comment:
Add test case to verify that it returns zero bytes for blockheight which is not stored.